addons/hr_homeworking/models/hr_homeworking.py

```diff
@@ -19,287 +14,14 @@ class HrEmployeeLocation(models.Model):
     work_location_type = fields.Selection(related="work_location_id.location_type")
     employee_id = fields.Many2one('hr.employee', default=lambda self: self.env.user.employee_id, required=True, ondelete="cascade")
     employee_name = fields.Char(related="employee_id.name")
-    weekday = fields.Integer(compute="_compute_weekday", store=True)
-    weekly = fields.Boolean(default=False)
-    start_date = fields.Date(string="Start Date")
-    today_next_date = fields.Date(compute="_compute_today_next_date")
+    date = fields.Date(string="Date")
     day_week_string = fields.Char(compute="_compute_day_week_string")
-    end_date_create = fields.Date(help="used in computations on employee location pop-up in calendar view.")
-    removed = fields.Boolean(default=False, help="User removed worklocation on one day that has a weekly location")
-    parent_default_homeworking_id = fields.Many2one("hr.employee.location", help="original weekly worklocation on a removed worklocation", ondelete="cascade")
-    child_removed_ids = fields.One2many("hr.employee.location", 'parent_default_homeworking_id')
-    current_location = fields.Boolean(default=True, help="Whether the record is the most recent weekly worklocation")
-    end_date = fields.Date(help="end date of a previous weekly worklocation (as opposed to current worklocation)")
 
     _sql_constraints = [
-        ('uniq_exceptional_per_day', 'unique(employee_id, weekly, start_date, current_location)', 'Only one default work location and one exceptional work location per day per employee.'),
+        ('uniq_exceptional_per_day', 'unique(employee_id, date)', _('Only one default work location and one exceptional work location per day per employee.')),
```

Comment:
SQL constrains messages are automatically exported in .pot files :)
```suggestion
        ('uniq_exceptional_per_day', 'unique(employee_id, date)', 'Only one default work location and one exceptional work location per day per employee.'),
```